policies-and-procedures/jellyfin-constitution.md

```diff
@@ -0,0 +1,155 @@
+
+2. The Project Leader is the final arbiter in any disputes or decisions raised from lower teams and persons where such a decision cannot be reached by more collaborative means within the teams themselves.
+
+3. The Project Leader shall generally take a hands-off approach to day-to-day project decisions, delegating responsibility for these to the lower teams and persons.
+
+4. The Project Leader shall also function as Release Manager for the core server, handling the coordination and creation of new releases of the Jellyfin server software. Creation of releases for clients or subcomponents is delegated to their respective Function Teams.
+
+5. The Project Leader must always be a member of both the Leadership Team and the Financial Committee.
+
+#### Procedure
+
+1. The first Project Leader (Joshua Boniface) was and is self-appointed, as the original driver of the fork from Emby.
+
+2. The Project Leader shall serve indefinitely at their discretion, but may resign at any time.
+
+3. Upon resignation or indisposal of the Project Leader, one or more candidates for the position shall be proposed by the Leadership Team. A vote shall then be held by all members of the Jellyfin Contrbutor Team to confirm or select the chosen candidate(s). The method of election shall be ranked voting instant runoff until one candidate has secured an absolute majority of the vote. In the event that only a single candidate is proposed, and this candidate does not gain a majority of votes amongst the Contributor Team, the outgoing Project Leader shall cast a deciding vote to confirm or deny the candidate outright. If the outgoing Project Leader does not confirm the candidate, the process shall begin again and the previous candidate is inelligible for further consideration.
+
+4. The Project Leader may at their discretion appoint a temporary Interim Project Leader to handle their duties due to short-term absense or for the duration of a Projet Leader election.
+
+### Leadership Team ("Core")
+
+#### Powers
+
+1. The Leadership Team is responsible for the day-to-day operations of the project and the handling of technical and managerial decisions for the project.
+
+2. The Leadership Team is the only group with "Merge" permissions for the core server repositories, and are responsible for the merging of reviewed and approved Pull Requests into those repositories. All code changes must thus require implicit or explicit approval of a member of the Leadership Team.
+
+3. The Leadership Team should at all times provide a unified front for decisions, working under the principle of democratic centralism - lively debate and disagreement within the team is encouraged during decisionmaking, but decisions should be accepted and promulaged by the team as a unified entity without public disagreement. If internal decisionmaking fails, the decision shall fall to the Project Leader instead.
```

Comment:
```suggestion
3. The Leadership Team should at all times provide a unified front for decisions, working under the principle of democratic centralism - lively debate and disagreement within the team is encouraged during decisionmaking, but decisions should be accepted and promulaged by the team as an unified entity without public disagreement. If internal decisionmaking fails, the decision shall fall to the Project Leader instead.
```

Reply:
The original is correct. The `u` in "unified" is pronounced `/ju:/` and `/j/` is a consonant sound, which means `a unified` is the correct form here.

One of those weird English exceptions :p